src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -272,46 +281,61 @@ def generate_businesses(self, pop_data: SimulantData) -> pd.DataFrame():
             }
         )
 
+        # Generate random employers
         pct_adults_needing_employers = 1 - known_employers["prevalence"].sum()
-        n_need_employers = np.round(n_working_age * pct_adults_needing_employers)
-        employee_counts = np.random.lognormal(
```

Comment:
Yikes, was this not previously using CRN at all?

Reply:
It wasn't!